src/Overview.tsx

```diff
@@ -51,15 +80,68 @@ export default function Overview() {
           battery_level: status.batteryLevel(),
           status: status.status() ?? '',
           location: status.location() ?? '',
+          is_active: true,
+          last_updated: 'now',
         },
       }));
     }, [])
   );
 
+  useEffect(() => {
+    (async () => {
+      const baseUrl = new URL(config.serverUrl);
+      baseUrl.protocol = window.location.protocol;
+      const url = new URL('robots', baseUrl).toString();
+      const res = await fetch(url, {
+        method: 'get',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+
+      let robotInfo = await res.json();
```

Comment:
you should check [`res.ok`](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) before continuing here